### DISCUSSION TITLE
How to Trigger a Function After Manually Changing the Target or Source of a Link?

### DISCUSSION THREAD

**Discussion by goyaljai1** [clientIO/joint#2786] [Q&A] [answered]
### Introduction

I'm working with links, and I would like to execute a function whenever a user manually changes the source or target of a link. This function should allow me to update specific attributes of the link after it has been modified. Is there a built-in way to detect these changes and trigger a function accordingly? Thank you.

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS, JointJS+

**Comment by kumilingus** [comment, marked as answer]
See [link:connect](https://docs.jointjs.com/api/dia/Paper/#connect) paper event.
